docs/content/file-formats.md

```diff
@@ -61,6 +67,45 @@ spec:
 
 [CredentialSet]: /operator/glossary/#credentialset
 
+## ParameterSet
+
+See the glossary for more information about the [ParameterSet] resource.
+
+The ParameterSet spec is the same schema as the ParameterSet resource in Porter.
+You can copy/paste the output of the `porter parameters show NAME -o yaml` command into the ParameterSet resource spec (removing the status section).
+
+In addition to the normal fields available on a [Porter Parameter Set document](/reference/file-formats/), the following fields are supported:
```

Comment:
Can you update the link to go directly to the correct section on the page?

Reply:
Whoops copy paste error :)